client/src/components/Workflow/Run/services.js

```diff
@@ -31,3 +31,22 @@ export async function invokeWorkflow(workflowId, invocationData) {
     const response = await axios.post(url, invocationData);
     return response.data;
 }
+
+/**
+ * Request tool step data.
+ *
+ * @param {String} workflowId - (Stored?) Workflow ID to fetch data for.
+ */
+export async function getTool(toolId, toolVersion, toolInputs) {
```

Comment:
Maybe include the history_id here ? Without it the server-side trans.history is used, and we want to move away from that.

Reply:
Thanks!